scripts/transfer-doc-files.js

```diff
@@ -2,42 +2,107 @@ const path = require('path');
 const fs = require('fs');
 const { execSync } = require('child_process');
 
-const DOC_FILES_DIRECTORY = path.join(__dirname, '../docs/static/components')
+const DOC_FILES_DIRECTORY = path.join(__dirname, '../docs/static/components');
+const DOC_FILES_CONTENT_DIRECTORY = path.join(__dirname, '../docs/content/components');
+
+const frontMatterTemplate = (componentName) => {
+    return `---
+title: "${componentName.charAt(0).toUpperCase() + componentName.slice(1)}"
+date: 2020-10-08T14:00:45Z

```

Comment:
Tried it when we first tested HUGO.  Changing the data always resulted in not showing the dec file. Didn't want to spent time debugging it then and since it's not that important, I don't want to waste time now as well. But I'll create a task to check it, because it would be nice if the date it correct. - https://coherent-labs.atlassian.net/browse/COH-15207